docs/src/modules/components/HighlightedCode.js

```diff
@@ -1,29 +1,72 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import prism from '@material-ui/markdown/prism';
+import FileCopyIcon from '@material-ui/icons/FileCopy';
+import IconButton from '@material-ui/core/IconButton';
+import copy from 'clipboard-copy';
+import Snackbar from '@material-ui/core/Snackbar';
 import MarkdownElement from './MarkdownElement';
+import { useTranslate } from '../utils/i18n';
 
 const HighlightedCode = React.forwardRef(function HighlightedCode(props, ref) {
-  const { code, language, ...other } = props;
+  const { code, language, isCopyButtonEnabled, ...other } = props;
```

Comment:
I suspect we want to enable it by default since most usages are for inline code. Only the demo would disable it, right?

Reply:
Expect demos, all inline code blocks will have copy button. On the other hand some pages using Highlighted code components will have copy button too, because I changed that manually, not demo component which we talked about. 

I suspected about that too, but when I saw the github improvement about these blocks - they did for all github markdown view - I though it could not be problem. 

Only migration blocks may be problem, you know migration inline blocks only show that the differences but we are adding copy button too. It seems a little strange. But github has same strange behaviour too like below. They are adding copy button all inline code blocks directly. 

```
 warning = {
-  main: orange[500],
+  main: "#ED6C02", // orange[400] in "dark" mode

-  light: orange[300],
+  light: orange[500], // orange[300] in "dark" mode

-  dark: orange[700],
+  dark: orange[900], // orange[700] in "dark" mode
 }
```